storybook/stories/playground/box/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { registerCoreBlocks } from '@wordpress/block-library';
+import {
+	BlockEditorProvider,
+	BlockCanvas,
+	BlockTools,
+} from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import editorStyles from '../editor-styles';
+import './style.css';
+
+export default function EditorBox() {
+	const [ blocks, updateBlocks ] = useState( [] );
+
+	useEffect( () => {
+		registerCoreBlocks();
+	}, [] );
+
+	return (
+		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+		<div
+			className="editor-box"
+			onKeyDown={ ( event ) => event.stopPropagation() }
```

Comment:
Why?

Reply:
Because we have some global keyboard shortcuts that conflict playground's shortcuts (try typing in the editor). Feels like we may want to call "preventDefault" when typing random characters in the editor but it's out of scope here.

Reply:
Adding this to the code example doesn't feel right. Should maybe move to a general playground template